Update Obsidian Sync and third-party services.md (#379)
obsidianmd/obsidian-docs: en/Obsidian Sync/Obsidian Sync and third-party services.md

> [!important] Files On-Demand
> Some cloud storage services, such as OneDrive, allow you to only download files when you use them and later remove them locally to free up space. Since the files are no longer available locally, Obsidian Sync believes they have been deleted and removes them from your remote vault.
>
> To use Obsidian Sync together with Files On-Demand and similar features, make sure to configure the service to always keep the files on the device.

# Removing your vault from a third-party sync service
If you are using Obsidian Sync and realize that you've set up your vault in a folder synced by a third-party service (such as Dropbox, OneDrive, or iCloud Drive), you can use the following steps to move the vault to a safer location.

## Desktop (macOS/Windows/Linux)
First, try the "Move vault" feature in the [[Vault switcher]]. 
1. Open the vault switcher (in the default theme, this is represented by a vault icon in the bottom-left of the window).
2. Select the three dots next to the vault you wish to move. 
3. Choose a new location for the vault on your file system.

> If that doesn't work (e.g., you're moving across drives), do the following. (This might look like a lot of steps, but it's really just "move your vault folder, then open it in Obsidian as a 'new' vault.")

Reviewer: ```suggestion
### Option 2: Move your vault manually
```

> 1. Make backups. (E.g., copy your vault folder somewhere else that you won't be touching throughout the rest of this operation.)  

Reviewer: Maybe we should put a "Back up your vault" callout under the `## Remove your vault ...` heading. Feels like a good thing to do regardless of the option they choose, and a good warning before they pick an option.

2. Quit Obsidian.  
3. Cut and paste (or move, e.g., via drag and drop) your vault folder from the old location to your newly-selected vault location (Reminder: do not put it in a folder that is being synced by any other services.)
4. Check and make sure it has all your stuff, otherwise go back to (3).  
5. Relaunch Obsidian.  

> 6. Open the Vault Picker, and click "Open folder as vault"  

Reviewer: ```suggestion
6. Open the Vault Picker, and then select **Open folder as vault**.
```

Vault picker or Vault switcher?

7. Navigate to your vault's new location and choose your vault folder.  
8. Check and make sure it looks the same. You might have to re-enable community plugins (Settings → Community Plugins → Turn restricted mode off.)  

> 9. Re-set up Sync.

Reviewer: ```suggestion
9. Set up Obsidian Sync again.
```

### iOS
1. Make a backup of your vault somewhere (just in case).
2. On your Obsidian device, create a new, temporary vault, and make sure "Save in iCloud Drive" is not enabled. (This is to make sure the app container folder we'll use in the next step exists. You'll delete this fake vault in a moment.)

> 3. Force quit the Obsidian app on all iOS devices and quit the app on any desktops. (This is to prevent Sync from doing anything while we move our files.)

Reviewer: ```suggestion
3. Force quit the Obsidian app on all your devices to prevent Sync from performing any operations while you move the files.
```

4. On your iOS device, go into the Files app, navigate to iCloud Drive → Obsidian, and long-press on your vault's folder. Select Move, and navigate to On My iPhone → Obsidian. (You should see the fake vault you created in step 2 here.) Press "Copy."
5. Once the copying is finished: still in the Files app, navigate back to iCloud Drive → Obsidian. Delete your vault folder there.

> 6. Open the Obsidian app. You should be presented with the vault picker and your vault should now have a vault icon instead of a cloud, indicating it is no longer on iCloud Drive.

Reviewer: ```suggestion
The next time you open Obsidian and the Vault switcher, your vault now has a vault icon instead of a cloud, indicating it's no longer on iCloud Drive.
```

> Some cloud storage services, such as OneDrive, allow you to only download files when you use them and later remove them locally to free up space. Since the files are no longer available locally, Obsidian Sync believes they've been deleted and removes them from your remote vault.
>
> To use Obsidian Sync together with Files On-Demand and similar features, make sure to configure the service to always keep the files on the device.

## Remove your vault from a third-party sync service

If you're using Obsidian Sync and realize that you've set up your vault in a folder synced by a third-party service, you can use the following steps to move the vault to a safer location.

## Desktop

### Option 1: Move your vault using the Vault switcher

1. In the bottom-left corner, select **Open another vault** (vault icon).
2. Next to the vault you want to move, select **More options** (three dots icon). 
3. Choose a new location for the vault on your file system.

### Option 2: Move your vault manually
  

> 1. Make backups. (E.g., copy your vault folder somewhere else that you won't be touching throughout the rest of this operation.)  

Reviewer: ```suggestion
1. Make backups of your vault. Copy your vault folder another location that you won't touch throughout the rest of this operation.
```

2. Quit Obsidian.  

> 3. Cut and paste (or move, e.g., via drag and drop) your vault folder from the old location to your newly-selected vault location (Reminder: do not put it in a folder that is being synced by any other services.)

Reviewer: ```suggestion
3. Cut and paste (or move using drag and drop) your vault folder from the old location to your newly-selected vault location Don't put it in a folder that is being synced by another service.
```

> 4. Check and make sure it has all your stuff, otherwise go back to (3).  

Reviewer: ```suggestion
4. Make sure your backup folder contains your vault.  
```

5. Relaunch Obsidian.  
6. Open the Vault switcher, and then select **Open folder as vault**.
7. Navigate to your vault's new location and choose your vault folder.  
8. Check and make sure the vault looks the same. You might have to re-enable community plugins under **Settings → Community Plugins → Turn restricted mode off**.)  
9. Set up Obsidian Sync again.

### iOS

> 1. Make a backup of your vault somewhere (just in case).

Reviewer: ```suggestion
1. Make a backup of your vault.
```

2. On your device, create a new vault and disable **Save in iCloud Drive**.
3. Force quit the Obsidian app on all your devices to prevent Sync from performing any operations while you move the files.
4. On your iOS device, open the Files app.

> 5. Under **iCloud Drive** → **Obsidian**, long-press on the vault folder and then select **Move**.

Reviewer: ```suggestion
5. Under **iCloud Drive → Obsidian**, long-press on the vault folder and then select **Move**.
```
